### PR TITLE
Add voting delay and period

### PR DESCRIPTION
**Pull request by samuveth**
Fixes https://github.com/snapshot-labs/snapshot/issues/709

### CODE REVIEW

**Comment by samuveth** [comment]
Requires https://github.com/snapshot-labs/snapshot.js/pull/390

File: src/helpers/utils.ts [hunk 1]
```diff
@@ -134,3 +134,13 @@ export function n(number, format = '(0.[00]a)') {
 export function ms(number) {
   return format(number * 1e3);
 }
+
+export function calcFromSeconds(value, unit) {
```
Comment (midgerate) [inline src/helpers/utils.ts:138]: Its bit hard to undertsand. 

What are you trying to calculate from seconds? Days? Shouldnt it be division?

Comment (samuveth) [reply src/helpers/utils.ts:138]: @midgerate the unit, it is doing a division.

File: src/views/SpaceSettings.vue [hunk 2]
```diff
@@ -497,6 +520,38 @@ watchEffect(async () => {
               </div>
             </div>
           </Block>
+          <Block :title="$t('settings.voting')">
+            <UiInput v-model="votingDelay" :number="true" placeholder="12">
```
Comment (bonustrack) [inline src/views/SpaceSettings.vue:524]: Can you use `e.g. 1` days  and  `e.g. 5` days as placeholder?
Instead of `12` hours  and  `5` days

**Pull request merged**